tests/NewCypress/cypress/e2e/regression/pagination.cy.ts

```diff
@@ -0,0 +1,82 @@
+import paginationPage from "../../pages/paginationPage";
+
+describe('Pagination Tests', () => {
+
+
+    beforeEach(() => {
+        cy.login()
+        cy.visit('/search?keywords=west')
+    });
+
+    it('Should display multiple pagination buttons', () => {
+        paginationPage.getTotalPaginationButtons().should('be.greaterThan', 1);
+    });
+
+    it('Should navigate to a specific page when a page number is clicked', () => {
+        const targetPage = 3;
+
+        paginationPage
+            .getResults()
+            .then(initialResults => {
+                const initialFirstResultText = initialResults
+                    .first()
+                    .text();
+
+                paginationPage
+                    .clickPageNumber(targetPage);
+
+                paginationPage
+                    .getResults()
+                    .first()
+                    .should('not.have.text', initialFirstResultText);
+            });
```

Comment:
This test ensures that we're on a different page to the first result page but not necessarily that we're on the third result page. There's a few different ways to tell which page we're actually on that we could test:
- the url ends in `/search?keywords=west&pagenumber=3`
- the page title is `Search (page 3 of 3)`
- the current page is highlighted in the pagination bar at the bottom (the element has `govuk-pagination__item--current`)

Reply:
We could fold this logic into the pageobject model - `paginationPage.checkCurrentPageIs(3)` which would simplify all the tests and allow us to easily test multiple navigations

e.g.
```
paginationPage.checkCurrentPageIs(1)

paginationPage.clickPageNumber(3)
paginationPage.checkCurrentPageIs(3)

paginationPage.clickPageNumber(2);
paginationPage.checkCurrentPageIs(2)
```

Reply:
Now implemented the above functionality